applications/RomApplication/tests/compressible_potential_test_files/simple_airfoil.mdpa

```diff
@@ -0,0 +1,2810 @@
+Begin ModelPartData
```

Comment:
I see this mdpa is not so big but in the CompressibleApp we have this mdpa:

https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CompressiblePotentialFlowApplication/tests/naca0012_small_mdpa/naca0012_small.mdpa

Which is much smaller, if it's not too much work and it fits your needs you can consider using it instead.

Reply:
Yep, good point. Done.